src/components/Profile.jsx

```diff
@@ -20,10 +20,10 @@ class Profile extends React.Component {
 
 Profile.propTypes = {
 	user: PropTypes.shape({
-		images: PropTypes.array,
+		images: PropTypes.arrayOf(PropTypes.number).isRequired,
```

Comment:
Not a number, we're accessing the _index_ 0. Each image is an object, that looks like:
```
{
  small: "http://...jpg",
  large: "http://...jpg",
}
```
so the propType would look like:
```
PropTypes.arrayOf(PropTypes.shape({
  small: PropTypes.string,
  large: PropTypes.string,
}).isRequired
```